src/mame/misc/astrcorp.cpp

```diff
@@ -234,25 +384,32 @@ void astrocorp_state::video_start()
 
 void astrocorp_state::draw_sprites(bitmap_ind16 &bitmap, const rectangle &cliprect)
 {
-	u16 *source = m_spriteram;
-	u16 *finish = m_spriteram + m_spriteram.bytes() / 2;
+	const u16 *source = m_spriteram;
+	const u16 * const finish = m_spriteram + m_spriteram.bytes() / 2;
```

Comment:
You can use `m_spriteram.length()` to get the size of the share in elements rather than bytes, making the division here unnecessary.

Reply:
Changed to sue length().